<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModbusClient</class>
 <widget class="QMainWindow" name="ModbusClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModbusClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Modbus TCP клиент</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Порт подключения:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="lbResponse">
        <property name="text">
         <string>ответ...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Имя сервера:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lePort">
        <property name="text">
         <string>502</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbHostName"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Ответ сервера:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbSendRequest">
        <property name="text">
         <string>Отправить запрос</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbQuit">
        <property name="text">
         <string>Отключиться</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>282</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
